### ISSUE TITLE
Fix deberta doc example

### ISSUE THREAD

**Issue by yisz** [relari-ai/continuous-eval#16]
In the deberta answer score example, it shows the logits not the normalized probiliities.
